website/blog/2025-04-20-vs-code-with-podman-desktop.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Supercharge Your Container Development in VS Code with Podman and Podman Desktop
+description: How to develop apps with VS Code and Podman Desktop
+authors: [firewall]
+tags: [podman-desktop, podman, vscode, docker, extension, development, container, tools]
+hide_table_of_contents: false
+---
+
+Developing containerized applications can sometimes feel complex, but with the right tools, it can be a smooth and efficient process. In this blog post, we'll explore how to leverage the power of Visual Studio Code (VS Code) together with Podman and Podman Desktop to streamline your container development workflow. We'll cover setting up the necessary tools and using the VS Code extensions for a seamless experience.
+
+## VS Code: Your IDE for Container Development
+
+VS Code is a popular and versatile code editor that can be extended to enhance its functionality. For container development, several excellent extensions integrate seamlessly with Podman.
+
+## Prerequisites
+
+Before we begin, ensure you have the following installed:
+
+- **Podman:** Follow the installation instructions on the [official website](https://podman.io/).
+- **Podman Desktop:** Download and install Podman Desktop from the [official website](https://podman-desktop.io/downloads).
+- **Visual Studio Code:** Download and install VS Code from the [official website](https://code.visualstudio.com/download).
+
+## **VS Code Extensions**
+
+To integrate VS Code with Podman, we have 2 extensions as options:
+
+1. [**Microsoft’s Container Tools extension**](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-containers)**:** The "Container Tools" extension provides excellent support for container-related tasks, including building images, managing containers, and working with Containerfiles and Dockerfiles.
+2. [**Pod Manager**](https://marketplace.visualstudio.com/items/?itemName=dreamcatcher45.podmanager): This extension was created by one of the members of our community and is completely open source. It is designed to help you manage Podman containers, images, volumes, and networks directly from the VS Code interface.
+
+Let's install them one by one.
```

Comment:
Why would someone install the two extensions?
Wouldn't the user pick the one they prefer? 

Maybe, you could say something like "While it's unlikely that you would use both extensions simultaneously, comparing them will help you understand the strengths and weaknesses of each, ultimately making it easier to select the one that best fits your needs. So for this blog post, we'll install them one by one."